src/integrationTest/java/uk/gov/hmcts/reform/wataskmanagementapi/cft/query/ia/CftQueryServiceITTest.java

```diff
@@ -1022,10 +1025,10 @@ private static Stream<TaskQueryScenario> withAllGrantTypesHappyPath() {
                     "8d6cc5cf-c973-11eb-bdba-0242ac111024", "1623278362431024",
                     "8d6cc5cf-c973-11eb-bdba-0242ac111003", "1623278362431003",
                     "8d6cc5cf-c973-11eb-bdba-0242ac111023", "1623278362431023",
-                    "8d6cc5cf-c973-11eb-bdba-0242ac111006", "1623278362431006",
                     "8d6cc5cf-c973-11eb-bdba-0242ac111007", "1623278362431007",
                     "8d6cc5cf-c973-11eb-bdba-0242ac111008", "1623278362431008",
-                    "8d6cc5cf-c973-11eb-bdba-0242ac111009", "1623278362431009"
+                    "8d6cc5cf-c973-11eb-bdba-0242ac111009", "1623278362431009",
```

Comment:
As we now have the AC's locked in for the sort, can we embellish taskDetail assertion here to include the dueDate. So it shows the date on the task as Desc.